execution/gethexec/node.go

```diff
@@ -185,7 +185,12 @@ func CreateExecutionNode(
 		if err != nil {
 			return nil, err
 		}
-		txPublisher = sequencer
+		if config.Sequencer.EnableEspressoFinalityNode {
+			espressoFinalityNode := NewEspressoFinalityNode(execEngine, seqConfigFetcher)
```

Comment:
Can we add a check that "sovereign sequencer flag" is not enabled when finality node is running?